### ISSUE TITLE
Upgraded to Postman 7.8 - LOST ALL MY COLLECTIONS!

### ISSUE THREAD

**Issue by oliviamoncayo** [a85/POSTMan-Chrome-Extension#7418]
Upgraded to Postman 7.8. Opened the application, created my account. I can see all my existing collections listed under My Workspace. 
I create a new collection, but it is read-only. I create a new request, it is read-only.
I then created a new workspace, gave it a name. I can create new collections there without a problem.
I switch back to My Workspace - all my collections I saw a moment ago are gone!

Anywhere I look for my previous work, just shows me the collection I created in the workspace I just created. My Workspace is completely empty.

YEARS of work - GONE!!!

Where can I look for my work, to recover it?

**Comment by xk0der** [comment]
Sorry about that Olivia, that's not the kind of experience we want our users to have.

Please do "Submit a ticket" to open a support request from here: https://www.getpostman.com/support 

In that support request provide us with your email ID that you used to create your account so that we may check to see how we may help you.

**Comment by xk0der** [comment]
While you log a support request with us, you mentioned you created a new account
> created my account

Is it possible you were using another email ID earlier? And then created a new account?

Also, do check here: https://app.postman.co/me/collections and see if any of your missing collections show up.

Plus check the trash as well: https://learning.getpostman.com/docs/postman/collections/managing_collections/#recover-a-collection

**Comment by oliviamoncayo** [comment]
I had version 5.5.3. If I had an earlier account, I do not recall. It still does not explain why with the "new" account I was able to see all my work when I first opened the application (under My Workspace), but when creating a new workspace (XYZ), all my work from My Workspace just disappeared.
The collections page did not show my previous work.
What really saved me was https://stackoverflow.com/questions/52303019/where-does-postman-save-your-local-collections-on-a-mac - fortunately right before upgrade I had opened the original Postman 5.5.3. It took a local backup automatically, and I was able to recover my work from there.
That was definitely some very scary and frustrating couple of hours. A very disappointing upgrade experience.
